src/main/java/mate/academy/springbootintro/repository/book/BookSearchParam.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springbootintro.repository.book;
+
+public enum BookSearchParam {

```

Comment:
What you need to do is to add field representing paramater's value:
```
TITLE("title"),
AUTHOR("author"),
PRICE("price");

@Getter
private String value;

public BookSearchParam(String value) {
  this.value = value;
}

public static BookSearchParam valueOf(String s) {
  for(BookSearchParam p : values() {
    if(p.getValue().equals(s)) {
      return p;
    }
    //throw IllegalArgumentException
  }
}
```

then, you can use `BookSearchParam.TITLE.getValue()` to get param value